plugins/modules/gitlab_project.py

```diff
@@ -162,10 +162,20 @@
     type: bool
     default: false
     version_added: "4.0.0"
+  issues_access_level:
+    description:
+      - V(private) means that accessing issues tab is allowed only to project members.
+      - V(disabled) means that accessing issues tab is disabled.
+      - V(enabled) means that accessing issues tab is enabled.
+      - O(issues_access_level) and O(issues_enabled) are mutually exclusive.
+    type: str
+    choices: ["private", "disabled", "enabled"]
+    version_added: "9.4.0"
   issues_enabled:
     description:
       - Whether you want to create issues or not.
       - Possible values are true and false.
```

Comment:
While you're at it, please remove this. `type: bool` already implies that `true` and `false` are valid values, and on the docsite the choices `true` and `false` are shown next to the option :)
```suggestion
```